doc/api/tty.md

```diff
@@ -2,13 +2,19 @@
 
     Stability: 2 - Stable
 
-The `tty` module houses the `tty.ReadStream` and `tty.WriteStream` classes. In
-most cases, you will not need to use this module directly.
+The `tty` modules provides the `tty.ReadStream` and `tty.WriteStream` classes.
+In most cases, it will not be necessary to use this module directly. However,
```

Comment:
Directly creating `tty` streams like shown above is, afaik, the only way to get a `process.stdin`-like `tty.ReadStream` (and correspondingly for `stdout`/`stderr`) when the standard file descriptors are used for redirection.

I don’t know if there are many Node.js applications that make use of this, but it’s definitely a meaningful thing to do; it’s what e.g. `sudo` does so that the password prompt doesn’t interfere with the normal I/O.